terraform/main.tf

```diff
@@ -167,6 +167,7 @@ module "batch_data_lake_writer" {
   use_ephemeral_storage    = true
   # SPOT is actually the default, this is just a placeholder until GTC-1791 is done
   launch_type              = "SPOT"
+  instance_types           = ["r5d.4xlarge", "r5d.8xlarge", "r5d.12xlarge", "r5d.16xlarge", "r5d.24xlarge", "r5ad.4xlarge", "r5ad.8xlarge", "r5ad.12xlarge", "r5ad.16xlarge", "r5ad.24xlarge", "c5d.12xlarge", "c5d.18xlarge", "c5d.24xlarge"]
```

Comment:
👋 The AMD instances are `r5ad`, right? The others are still intel, @dmannarino?

Reply:
Exactly.